src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -1,37 +1,87 @@
+/**
+ * @jest-environment jsdom
+ */
+
 import { library } from '@fortawesome/fontawesome-svg-core'
-import { faCoffee, faCircle } from '../__fixtures__/icons'
+import { faCoffee, faCircle, faAlien } from '../__fixtures__/icons'
 import { compileAndMount, mountFromProps } from '../__fixtures__/helpers'
 import FontAwesomeIcon from '../FontAwesomeIcon'
 
 beforeEach(() => {
-  library.add(faCoffee, faCircle)
+  library.add(faCoffee, faCircle, faAlien)
 })
 
-test('using array format', () => {
-  const wrapper = mountFromProps({ icon: ['fas', 'coffee'] })
-
-  expect(wrapper.element.tagName).toBe('svg')
-  expect(wrapper.element.classList.contains('fa-coffee')).toBeTruthy()
+afterEach(() => {
+  library.reset()
 })
 
-test('using string format', () => {
-  const wrapper = mountFromProps({ icon: 'coffee' })
+const version6 = process.env.npm_lifecycle_event.includes('test.6');
 
-  expect(wrapper.element.tagName).toBe('svg')
-  expect(wrapper.element.classList.contains('fa-coffee')).toBeTruthy()
-})
+const testIf = (condition, ...args) =>
+  condition ? test(...args) : test.skip
 
-test('missing icon', () => {
-  const wrapper = mountFromProps({ icon: ['fas', 'noicon'] })
+describe('icons are showing', () => {
+  test('using array format, short prefix and short icon name', () => {
+    const wrapper = mountFromProps({ icon: ['fas', 'coffee'] })
 
-  expect(wrapper.element.tagName).toBeFalsy()
-})
+    expect(wrapper.element.tagName).toBe('svg')
+    expect(wrapper.element.classList.contains('fa-coffee')).toBeTruthy()
+  })
+
+  testIf(version6, 'using array format, short prefix and long icon name', () => {
```

Comment:
`testIf` will only run when `npm run test.6` is running; will skip when `npm run test.5` is running.